src/Parse/HttpClients/ParseStream.php

```diff
@@ -61,19 +61,16 @@ public function get($url)
     {
         try {
             // get our response
-            $response = file_get_contents($url, false, $this->stream);
+            $response = $this->getFileContents($url, false, $this->stream);
```

Comment:
This is a `ci` PR but it does some changes in source files; do these changes fix something? Then it would be a `fix` PR.

Reply:
They don’t fix anything, more like a refactor